include/grpc++/client_context.h

```diff
@@ -130,24 +148,57 @@ class ClientContext {
   ClientContext();
   ~ClientContext();
 
-  /// Create a new ClientContext that propagates some or all of its attributes
+  /// Create a new \a ClientContext according to \a options (\see
```

Comment:
Create a new ClientContext as a child of an incoming server call, according to ...